pulp_2to3_migration/app/plugin/content.py

```diff
@@ -235,7 +235,7 @@ def get_remote_by_importer_id(importer_id):
                 select_extra.append('pulp2content__pulp2_repo')
 
             pulp_2to3_detail_qs = pulp_2to3_detail_qs.select_related(*select_extra)
-            for pulp_2to3_detail_content in pulp_2to3_detail_qs.iterator():
+            for pulp_2to3_detail_content in pulp_2to3_detail_qs.iterator(chunk_size=800):
```

Comment:
out of curiosity, why 800?

Reply:
No particular reason, other than that it's smaller than 2000 but big enough to be worthwhile.  @goosemania was seeing some memory consumption issues and we were speculating about the causes - this might *not* be the problem, but it's worth trying since it's so easy.

Reply:
sounds good, I am probably biased by preferring having round number like 500,1000,2000,5000, etc <--- this is not a technically backed answer lol

Reply:
It's as equally technically-backed as mine :)